src/nupic/regions/TMRegion.hpp

```diff
@@ -90,6 +90,10 @@ class TMRegion : public RegionImpl {
 
 private:
   Dimensions columnDimensions_;
+
+  // Note: to avoid deserialization problems due to differences in 
+  //       how compilers deal with structure padding, do not allow
+  //       any member to span an 64bit (8byte) boundary.
```

Comment:
is this really a problem? if so, can serialization do an implicit check and warn on such condition?